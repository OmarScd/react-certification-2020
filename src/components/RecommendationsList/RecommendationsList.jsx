import React from 'react';
import styled from 'styled-components';

const Container = styled.div`
  display: flex;
  height: 100%;
  flex-direction: column;
  overflow-y: scroll;
`;

const RecommendationsList = ({ children }) => {
  return (
    <Container>
      <h2>Related Videos</h2>
      {children}
    </Container>
  );
};

export default RecommendationsList;
