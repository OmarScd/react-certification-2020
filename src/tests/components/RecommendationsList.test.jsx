import React from 'react';
import { render, cleanup } from '@testing-library/react';
import RecommendationsList from '../../components/RecommendationsList';
import ThemeProvider from '../../providers/Theme';

describe('RecommendationsList', () => {
  afterAll(cleanup);

  it('renders correctly', () => {
    const { container } = render(
      <ThemeProvider>
        <RecommendationsList />
      </ThemeProvider>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
