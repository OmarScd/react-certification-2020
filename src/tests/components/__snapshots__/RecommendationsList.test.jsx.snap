// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecommendationsList renders correctly 1`] = `
<div
  class="sc-bdnylx bMIijM"
>
  <h2>
    Related Videos
  </h2>
</div>
`;
